packages/payload/src/utilities/configToJSONSchema.ts

```diff
@@ -67,6 +67,30 @@ function generateEntitySchemas(
   }
 }
 
+function generateAuthEntitySchemas(entities: SanitizedCollectionConfig[]): JSONSchema4 {
```

Comment:
can we not do another loop here, and instead, convert the existing function that loops over collections to return auth-enabled collections?